api/src/pcapi/core/users/models.py

```diff
@@ -735,7 +735,9 @@ def has_partner_page(self) -> bool:
             sa.select(1)
             .select_from(UserOfferer)
             .join(Venue, UserOfferer.offererId == Venue.managingOffererId)
+            .join(User, UserOfferer.userId == User.id)
```

Comment:
```suggestion
            .join(User, UserOfferer.user)
```
ça fait la m^me jointure (le même SQL est généré) mais c'est plus élégant et lisible :)